### PR TITLE
Clarify that sealing check is not needed on pcc

### PR DESCRIPTION
**Pull request by andresag01**
Fixes https://github.com/riscv/riscv-cheri/issues/85

### CODE REVIEW

**Review by tariqkurd-repo — APPROVED** [review]

File: src/riscv-integration.adoc [hunk 1]
```diff
@@ -56,10 +56,14 @@ executed in addition to the checks already required by the base RISC-V ISA. A
 failing check causes a CHERI exception.
 
 * The tag must be set
-* The capability must not be sealed
 * The capability must grant execute permission
 * All bytes of the instruction must be in bounds
 
+NOTE: Operations that update <<pcc>>, such as changing privilege or executing
+jump instructions, unseal capabilities prior to writing. Therefore,
```
Comment (francislaus) [inline src/riscv-integration.adoc:63]: While I think we all believe that PCC can never be sealed, we have no proof of this. I am not sure whether it is the right way forward to claim that the sealing check can be omitted without having a proof. Checking that PCC can never be sealed seems to be a perfect case for a Sail proof. I would suggest to leave the check in until we have a proof and change the note such that it states our beliefs.

Comment (arichardson) [reply src/riscv-integration.adoc:63]: I agree, while I'm pretty sure this cannot happen, I'd leave the requirement above and just add this note.

Comment (andresag01) [reply src/riscv-integration.adoc:64]: No problem. I left the note and put the check back in.

**Review by arichardson — APPROVED** [review]
LGTM with some cautioning wording. @francislaus what do you think?

**Comment by francislaus** [comment]
LGTM with the suggestions from @arichardson.

**Review by francislaus — APPROVED** [review]

**Pull request merged**